relay.go

```diff
@@ -109,41 +105,51 @@ func (r *RelayCmd) startRPC(ctx context.Context, backend *RelayBackend) {
 }
 
 type RelayBackend struct {
-	log              logrus.Ext1FieldLogger
-	engine           *EngineCmd
-	payloadIdCounter uint64
-	recentPayloads   *lru.Cache
+	log            logrus.Ext1FieldLogger
+	engine         *EngineCmd
+	recentPayloads *lru.Cache
 }
 
 func NewRelayBackend(log logrus.Ext1FieldLogger) (*RelayBackend, error) {
 	engine := &EngineCmd{}
 	engine.Default()
 	engine.LogCmd.Default()
-	engine.ListenAddr = "127.0.0.1:28546"
-	engine.WebsocketAddr = "127.0.0.1:28547"
+	engine.ListenAddr = "127.0.0.1:8550"
+	engine.WebsocketAddr = "127.0.0.1:8552"
 	cache, err := lru.New(10)
+
 	if err != nil {
 		return nil, err
 	}
-	return &RelayBackend{log, engine, 0, cache}, nil
+	return &RelayBackend{log, engine, cache}, nil
+}
+
+func (r *RelayBackend) RegisterValidatorV1(ctx context.Context, message types.RegisterValidatorRequestMessage, signature hexutil.Bytes) (*string, error) {
+	res := "OK"
+	return &res, nil
```

Comment:
This functions needs to correctly register the validator, including verifying the signature.